guppy/heapy/test/test_Path.py

```diff
@@ -68,7 +68,7 @@ def f():
         self.chkpath(co, 3, '%s.co_consts[1]')
         # commented in notes Sep 27 2004
         relAttr = ('co_code', 'co_consts', 'co_names', 'co_varnames',
-                    'co_freevars', 'co_cellvars', 'co_filename', 'co_name')
+                'co_freevars', 'co_cellvars', 'co_filename', 'co_name')
```

Comment:
uhh, I mean the 'co_freevars' should be aligned with 'co_code'. Flake8 for example would complain:

```
$ cat test.py
relAttr = ('co_code', 'co_consts', 'co_names', 'co_varnames',
            'co_freevars', 'co_cellvars', 'co_filename', 'co_name')
$ flake8 --show-source test.py
test.py:2:13: E127 continuation line over-indented for visual indent
            'co_freevars', 'co_cellvars', 'co_filename', 'co_name')
            ^
```
```
$ cat test.py
relAttr = ('co_code', 'co_consts', 'co_names', 'co_varnames',
        'co_freevars', 'co_cellvars', 'co_filename', 'co_name')
$ flake8 --show-source test.py
test.py:2:9: E128 continuation line under-indented for visual indent
        'co_freevars', 'co_cellvars', 'co_filename', 'co_name')
        ^
```
```
$ cat test.py
relAttr = ('co_code', 'co_consts', 'co_names', 'co_varnames',
           'co_freevars', 'co_cellvars', 'co_filename', 'co_name')
$ flake8 --show-source test.py
```
(no errors in the last one)
See https://www.flake8rules.com/rules/E127.html & https://www.flake8rules.com/rules/E128.html

Reply:
Oh I see, my bad.